zml/exe.zig

```diff
@@ -210,11 +210,11 @@ pub const BaseExe = struct {
             .non_donatable_input_indices = &.{},
         }) catch unreachable;
 
-        for (events[0..sharding.num_partitions]) |e| {
-            if (e) |ev| {
-                ev.await_(self.platform.pjrt_api) catch unreachable;
-            }
-        }
+        // for (events[0..sharding.num_partitions]) |e| {
```

Comment:
please remove this, maybe leave a comment explaining we don't synchronize execution